### PR TITLE
Set drmhwc=false in mixins.spec for android_ia

### PR DESCRIPTION
**Pull request by mlofsted**
Signed-off-by: Marta Lofstedt <marta.lofstedt@intel.com>

### CODE REVIEW

File: androidia_64/mixins.spec [hunk 1]
```diff
@@ -7,7 +7,7 @@ product.mk: device.mk
 [groups]
 android_ia: default
 sepolicy: permissive
-graphics: android_ia(gen9+=true,hwc2=true,vulkan=true)
+graphics: android_ia(gen9+=true,hwc2=true,vulkan=true,drmhwc=false )
```
Comment (kalyankondapally) [inline androidia_64/mixins.spec]: Can you update the mixin changes and update Boardconfig configurations. Now, we default to drmhwc otherwise.

Comment (mlofsted) [inline androidia_64/mixins.spec]: Isn't the build system running the mixins-update if mixins.spec has changed?

Comment (kalyankondapally) [inline androidia_64/mixins.spec]: For now, its manual and needs to be done by us. Eventually CI will take care of this.

Comment (tpalli) [inline androidia_64/mixins.spec]: IMO we should hook mixins-update to run automatically on image build, right now it's pretty horrible as you cannot know if you should run it or not, maybe someone did changes, maybe not (?) only way to find out is to actually go and run it

Comment (kalyankondapally) [inline androidia_64/mixins.spec]: One doesnt need to run it. What ever is here (android_ia folder BoardConfig, device.mk etc) should be the version which anyone should worry about.

Comment (tpalli) [inline androidia_64/mixins.spec]: ok, so we should be manually editing device.mk and not touching mixins?

Comment (kalyankondapally) [inline androidia_64/mixins.spec]: Sorry, what I meant is this:
For now:
1) Update changes to mixin files.
2)run the mixin update and send the updated changes in android_ia as a separate pull request here.

Going forward:
1) We need to do only 1 and CI will go through the regression testing before doing the step 2 above.

From development pov, we should worry only about the changes in androidia_64 folder that they are stable and working.

**Comment by kalyankondapally** [comment]
Merged thanks

**Pull request closed without merging**